catalog/services_state_test.go

```diff
@@ -205,30 +205,31 @@ func Test_ServicesStateWithData(t *testing.T) {
 			})
 
 			Convey("Doesn't retransmit when the state is the same", func() {
-				state.Broadcasts = make(chan [][]byte, 1)
+				state.Broadcasts = make(chan [][]byte, 2)
 				state.AddServiceEntry(svc)
+
+				<-state.Broadcasts
```

Comment:
Probably comment here that we're doing this to make sure the goroutine runs. Otherwise it looks superfluous.